src/renderEpisodeCard.js

```diff
@@ -0,0 +1,45 @@
+import { renderElementIntoTarget, changeInnerHTML } from './genericFunctions';
+
+const blank = '';
+
+export function renderEpisodeCard() {
+    renderElementIntoTarget({
+        elementId: 'episodeCard',
+        elementType: 'div', 
+        content: blank,
+        elementClass: 'episodeCard', 
+        targetId: 'episodeBox' 
+    });
+    const cardInfo = ['episodeName', 'episodeCodeNumber', 'exibitionDate', 'characterList', 'characters']
+    for (let index = 0; index < cardInfo.length; index++) {
+        renderElementIntoTarget({
+            elementId: cardInfo[index],
+            elementType: 'p', 
+            content: blank,
+            elementClass: cardInfo[index], 
+            targetId: 'episodeCard' 
+        });
+    }
+}
+
+export function setOnclickAttribute(RaMQuery) {
+    const RaMResults = RaMQuery.data.data.episodes.results;
+    RaMResults.forEach(episode => document.getElementById(episode.id).onclick = renderCard.bind(this, RaMResults));
```

Comment:
Very nice!! 
Do you really need to bind `this` context into `renderCard` function? Whenever you see the keyword `this` you should think twice!
Since you are not really relying on `this` inside renderCard, you can just pass the null to the first argument.
But you can also do this:
``` js
RaMResults.forEach(episode => document.getElementById(episode.id).onclick = () => renderCard(RaMResults))
```

And avoid using `this`, `bind` and `call` keywords/functions